mod.conf

```diff
@@ -1 +1,4 @@
 name = chat_anticurse
+description = Kick player out when they type curse word in chat or using /me or /msg command. Checks for some English, Russian, and German curse words.
+optional_depends = default
```

Comment:
`default` isn't used anywhere in this mod, pretty sure this dependency is useless.

Reply:
The original mod author had `default` as an optional dependency, im not sure why?

Reply:
Tested the mod in devtest, default is not needed